01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+      <title>index</title>
+      <meta name="description" content="">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="style1-1.css">
+      <script src="1-1.js"></script>
+  </head>
+
+  <body onload="setTimeout (showonload,1000)">
+    <section id="hidden">
+      Hello world.
+    </section>
+
+    <button type="button" id="buttonJoke"onclick="joke ()">Joke</button>
+    <section id="sectionJoke">
+    </section>
+
+    <button type="button" id="buttonReusableJoke"onclick="ReusableJoke ()">ReusableJokeConfig</button>
+    <section id="sectionReusableJoke">
+    </section>
+
+    <hr></hr>
+    <button class="button" onclick="QSearch()">QSearch</button>
+    <div>
```

Comment:
I left it empty so the field for the FreeSearch is not on the same line that the QSearch and the list for both gets generated under the FreeSearch